tools/sim/bridge/metadrive/metadrive_process.py

```diff
@@ -111,10 +112,18 @@ def get_cam_as_rgb(cam):
         reset()
 
     if rk.frame % 5 == 0:
-      obs, _, terminated, _, info = env.step(vc)
+      _, _, terminated, _, _ = env.step(vc)
+      time_out = True if time.monotonic() - start_time >= time_done else False
+      out_of_lane = env.vehicle.on_broken_line or env.vehicle.on_yellow_continuous_line or env.vehicle.on_white_continuous_line or env.vehicle.crash_sidewalk
```

Comment:
any way to check "vehicle in lane" or another simple check? looks like we might have to keep adding conditions for this one